stagemonitor-requestmonitor/src/main/java/org/stagemonitor/requestmonitor/MonitoredRequest.java

```diff
@@ -34,6 +36,8 @@
 	 */
 	Object execute() throws Exception;
```

Comment:
I can remove the execute method if you want. I didn't want to start removing methods from interfaces in case I broke something elsewhere.